validation/bickley_jet/bickley_jet.jl

```diff
@@ -0,0 +1,186 @@
+ENV["GKSwstype"] = "nul"
+using Plots
+
+using Printf
+using Statistics
+using CUDA
+
+using Oceananigans
+using Oceananigans.Advection
+using Oceananigans.AbstractOperations
+using Oceananigans.OutputWriters
+using Oceananigans.Grids
+using Oceananigans.Fields
+using Oceananigans.Utils: prettytime
+
+using Oceananigans.Models.HydrostaticFreeSurfaceModels: HydrostaticFreeSurfaceModel, ExplicitFreeSurface
+using Oceananigans.Models.HydrostaticFreeSurfaceModels: VectorInvariant
+
+include("disk_time_series.jl")
+
+using .DiskTimeSerieses: DiskTimeSeries
+
+#####
+##### The Bickley jet
```

Comment:
If you wanted to be more quantiative in the comparisons, we could look at a geostrophic Bickley jet, and compare with the results of the linear stability theory.  My experience tells me that the one-layer growth rates should be very similar to the stratified barotropic problem.

Reply:
That's a nice idea --- does your paper have results for spheres or just Cartesian domains? Either way that would be a good way to go for this experiment since we can both perform the quantitative validation, as well as obtaining a qualitative validation of advection schemes / numerical dissipation in the nonlinear regime.

The reason I am using this specific experiment is because we've run the same problem innumerable times in the course of testing our DG code. The WENO results demonstrate how the WENO advection scheme adds just the right amount of numerical dissipation to keep the simulation smooth while preserving sharp gradients. Validating numerical schemes in a strongly nonlinear flow was the original primary purpose of this experiment.

Reply:
My paper only discusses the planar case but should be easy enough to do the spherical case, if there is interest.

Also, good  you started with something you know well.